### DISCUSSION TITLE
What's the right way to write a condition for GitHub Actions with a matrix?

### DISCUSSION THREAD

**Discussion by CatChen** [Mergifyio/mergify#1995] [Q&A] [answered]
Hi, I read the Mergify doc on GitHub Actions: https://docs.mergify.io/conditions.html#github-actions. It's still unclear to me how I can match GitHub Actions with a matrix. Here is an example PR I have: https://github.com/CatChen/browser-id/pull/232. Thanks!

**Comment by sileht** [comment, marked as answer]
Hi, in your case the final GitHub Action reported by GitHub API are `build (10.x)`, `build (11.x)`, `build (12.x)`, ...
